Framework/DataObjects/inc/MantidDataObjects/MDBoxBase.tcc

```diff
@@ -54,8 +54,7 @@ TMDE(MDBoxBase)::MDBoxBase(
 }
 
 //-----------------------------------------------------------------------------------------------
-/** Copy constructor. Copies the extents, depth, etc.
- * and recalculates the boxes' volume.
+/** Copy constructor. Copies the extents, volume, depth, etc.
  * @param box :: incoming box to copy.
  * @param otherBC :: if present, other (different from the current one) box
  * controller pointer
```

Comment:
Instead of the for loop, I'm tempted to change `extents` and `m_centroid` from a C array to `std::array` and use the copy constructor.